src/microbe_stage/editor/tooltips/SelectionMenuToolTip.cs

```diff
@@ -209,6 +209,19 @@ public void WriteOrganelleProcessList(List<ProcessSpeedInformation>? processes)
         processList.ProcessesToShow = processes;
     }
 
+    /// <summary>
+    ///   Sets the value of this tooltip's osmoregulation cost modifier.
+    /// </summary>
+    /// <param name="osmoregulationCost">The value to set the osmoregulation cost to.</param>
+    public void WriteOsmoregulationCost(float osmoregulationCost)
+    {
+        foreach (var modifier in modifierInfos)
+        {
+            if (modifier.Name == "osmoregulationCost")
+                modifier.ModifierValue = $"+{osmoregulationCost}";
```

Comment:
I think it would make more sense to make this osmoregulation cost modifier like how the mutation cost is set, i.e. so create a new property with a backing field and relevant method to update the value. Then you'll need to go through all the existing tooltips in the godot editor to set the right initial values. Also for display I think the osmoregulation should be rounded to at most 2-3 decimal places.

Reply:
Fortunately, the tooltips already had the right initial values in the Godot editor.